spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JSqlParserQueryEnhancer.java

```diff
@@ -441,7 +442,7 @@ public String getProjection() {
 
 		SelectBody selectBody = selectStatement.getSelectBody();
 
-		if (selectStatement.getSelectBody()instanceof SetOperationList setOperationList) {
+		if (selectStatement.getSelectBody() instanceof SetOperationList setOperationList) {
 
```

Comment:
Whereas I sympathize with the desire to introduce this space WHICH SHOULD BE HERE, our current tools, due to `getSelectBody()` (the parentheses) will take that space back out the next time I'm inside this class and I reformat it. Hence, I'd rather not introduce that churn.